### PR TITLE
feature/arcwell-398/server_production_build

### PR DESCRIPTION
**Pull request by hswope-arcweb**
Updated Dockerfile and package.json to support production server builds.
Fixed bug in PersonFactory.

### CODE REVIEW

File: packages/server/Dockerfile [hunk 1]
```diff
@@ -13,46 +13,60 @@ ARG PORT=3333
...
 
+# install dependencies --include=dev required to install dev dependencies if NODE_ENV=production
+RUN npm install --no-fund --include=dev
 
-#
+# run build
+RUN npm run build
+
+#remove ace
+RUN rm ./build/ace.js
```
Comment (timgetz) [inline packages/server/Dockerfile]: We will need ace for production.  e.g. to run migrations, and/or seed app data.  Does removing this, remove ace commands?

**Review by timgetz — CHANGES_REQUESTED** [review]
@hswope-arcweb Looks good, but please add some instructions in the README about building for production.

File: packages/server/Dockerfile (hunk 1 above)
Comment (hswope-arcweb) [reply packages/server/Dockerfile]: Fixed.

**Review by timgetz — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -250,6 +250,16 @@ The [quickstart instructions](#quickstart) above include details on how to
...
+## Building for Production
+
+The [quickstart instructions](#quickstart) and guide to this point have been
+focused on building a _development_ environment. Arcwell can be built for
+production-like environments (e.g., your hosted staging, QA/dev, or production
+targets).
+
+Learn more about generated environment-specific, production-like builds within
+the [production build docs](doc/builds.md).
+
```
Comment (mccolin) [inline README.md:385]: I took a crack at editing this paragraph to reflect the goal of showing we offer production-ready builds and linking out to a special `builds.md` with further instructions.

File: doc/builds.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+docker build --target production -t arcwell-production .
+```
+
+## Environment Variables
+When deploying with either one of these builds one would want to make sure to
+set the running environment's variables to values appropriate for production.
+
+See `packages/server/.env.example` for possible environment variables.
+
+
```
Comment (mccolin) [inline doc/builds.md:45]: I moved these notes from `packages/README.md` to here. Moving forward, the `/doc` path should hold most content like this. The intention of the `packages/README.md` is to server more as an index of the included packages, which will be especially useful as the set of packages grows.

**Review by mccolin — COMMENTED** [review]
This looks good to me, but I will defer to @timgetz for a final approval and merge with other `Dockerfile` work you may have underway.

**Review by timgetz — APPROVED** [review]

**Pull request merged**